### ISSUE TITLE
Support multiple ECS clusters

### ISSUE THREAD

**Issue by ejholmes** [remind101/empire#912]
We've talked about this internally. There could be benefits to supporting multiple ECS clusters (say different clusters for different "products").

This is pretty easy to support in the backend using the CloudFormation scheduler, we'd just need to figure out the UI/UX for it.

**Comment by phobologic** [comment]
Yeah - just a couple of the ideas we were tossing around for how to indicate which cluster a process or app should go on (we should figure out if it's whole app, or could be down to the process - my initial gut feeling says whole app for v1):

- Use an environment variable, ie: `emp set -a acme-inc EMPIRE_CLUSTER=ecs-cluster2`
- Use a flag to most commands `emp deploy -c ecs-cluster2` - in this case we should figure out which cluster it's already deployed to automatically, so you don't have to specify cluster every time.

Any other ideas?
